### PR TITLE
add support for presentation of multiple days results

### PR DESCRIPTION
**Pull request by xuyangit**

### CODE REVIEW

File: scripts/analyze.py [hunk 1]
```diff
@@ -35,12 +35,11 @@ def parseFile(fileName):
 
 if __name__ == '__main__':
     args = sys.argv
-    if(len(args) != 4):
+    if(len(args) < 4):
```
Comment (yueawang) [inline scripts/analyze.py:38]: Have you already checkin the call change in shell file??
Does this 4 mean results in last 4 days? if so, should we make it configurable??

**Review by yueawang — COMMENTED** [review]
Thanks Yang, just 1 comment.

File: scripts/analyze.py (hunk 1 above)
Comment (xuyangit) [reply scripts/analyze.py:38]: Actually, 4 means at least py source file, day 1 res path, day 2 res path, output file name should be configed. And for any days larger than 2, it's ok. Not limited to 4.

**Comment by xuyangit** [comment]
Just now has done a rebase. It seems commit log is messed. Will create a new pr.

**Pull request closed without merging**